### ISSUE TITLE
Ask -> How to load on vuepack?

### ISSUE THREAD

**Issue by donnygunawan93** [marcosmoura/vue-material#53]
I use this starter pack [here](https://github.com/egoist/vuepack)
I change this file to import vue-material [here](https://github.com/egoist/vuepack/blob/master/template/client/components/App.vue)

so the code like below, and the button not look like material design.

![untitled](https://cloud.githubusercontent.com/assets/10414213/20243420/01a54cce-a989-11e6-92d7-4c274d71324d.png)

**Comment by cleverlycoding** [comment]
You should cut the code in the script tag and paste it inside `client/app.js` like this:
```js

import Vue from 'vue'
import {sync} from 'vuex-router-sync'
import VueMaterial from 'vue-material'
import 'vue-material/dist/vue-material.css'
import App from './components/App'
import router from './router'
import store from './store'
sync(store, router)

Vue.use(VueMaterial)
Vue.material.theme.register('default', {primary: 'cyan', accent: 'pink'})

const app = new Vue({
  router,
  store,
  ...App
})

export {app, router, store}
```

**Comment by donnygunawan93** [comment]
Already, but still the same.

**Comment by cleverlycoding** [comment]
Can you share your code? I could take a look at it and maybe see what's wrong with it.

**Comment by marcosmoura** [comment]
I know that this is bad, but something is wrong with VuePack trying to load vue-material.css file. Every single css file pass through a *post-css* loader and something is going 'crazy'. 

You can try to force a specific loader (css-loader) manually on your app.js file, like this:

```
import 'css!vue-material/dist/vue-material.css'
```

If this doesn't work for your you can to add a link tag on your HTML:

``` html
<link rel="stylesheet" href="path/to/vue-material.css">
```

Tell me if the first solution works for you. If does not work I will find another solution.

Thank you!

**Comment by donnygunawan93** [comment]
@cleverlycoding  yes...still default install only... [here](https://github.com/donnygunawan93/vuetest)
@marcosmoura hmr showing black screen bundle failed
do you think this is related with this [issue](https://github.com/egoist/vuepack/issues/61)?

**Comment by wstam88** [comment]
I have the same issue

**Comment by marcosmoura** [comment]
I don't think that this issue is the problem. I will try to download VuePack and install Vue Material to see if I can fix this for you. :)

**Comment by marcosmoura** [comment]
Sorry about the delay to solve this. I was focusing on the new release.

I found the problem. It's not inside Vue Material project indeed. This is because of CSSModules. You can learn more about in this<a href="https://github.com/css-modules/css-modules"> link</a>. CSSModules it is a good thing but when enabled (by default) on VuePack was applying some rules against css that modifies them. And this was affecting the vue-material.css changing all the rules inside. So it was breaking all the single components inside. Bad uh?
You can have 3 options:
* Disable CSSModules in <a href="https://github.com/egoist/vuepack/blob/master/template/build/config.js#L25">build/config.js</a> file 
* Change the rule related to cssModules on the css-loader <a href="https://github.com/egoist/vuepack/blob/master/template/build/utils.js#L7">config</a>To something like that:
``` javascript
_.cssLoader = 'css-loader?-autoprefixer'
```
* Fill an issue on VuePack asking about this. Maybe @egoist have a better fix for you. :)

Thanks.

**Comment by egoist** [comment]
I've made CSS modules disabled by default in VuePack, now it should work without any hack. But there's also an alternative way to solve this when CSS modules is enabled:

```vue
<!-- import it using style tag in .vue file -->
<style src="vue-material/dist/vue-material.css"></style>
```

**Comment by marcosmoura** [comment]
Oh. That's great. I didn't event think about this style thing. And thanks for answering this so quickly. :D

**Comment by capttrousers** [comment]
FYI @marcosmoura  this issue comes up using generic webpack as well. your fix doing the following worked for me. 
`import 'css!vue-material/dist/vue-material.css';`

**Comment by marcosmoura** [comment]
That's great. I simplified the documentation. Will be available along with v0.4.0.
